Runtime/Scripts/DefaultMaterialGenerator.cs

```diff
@@ -124,6 +150,41 @@ public class DefaultMaterialGenerator : IMaterialGenerator {
                 Debug.LogWarning("Double sided shading is not supported!");
             }
 
+            //Stephen Gower - added code for Enabling and Disabling Keywords for various alpha types.
+            //I don't remember which models were problematic, but it seemed to be the case that
+            //transparency wasn't always working the way it should. Added the code below fixed some of these issues.
+            if (gltfMaterial.alphaModeEnum == AlphaMode.MASK) {
+                material.name = gltfMaterial.alphaModeEnum.ToString();
+                material.SetOverrideTag("RenderType", "TransparentCutout");
+                material.SetInt("_SrcBlend", (int)UnityEngine.Rendering.BlendMode.One);
+                material.SetInt("_DstBlend", (int)UnityEngine.Rendering.BlendMode.Zero);
+                material.SetInt("_ZWrite", 1);
+                material.EnableKeyword("_ALPHATEST_ON");
+                material.DisableKeyword("_ALPHABLEND_ON");
+                material.DisableKeyword("_ALPHAPREMULTIPLY_ON");
+                material.renderQueue = (int)UnityEngine.Rendering.RenderQueue.AlphaTest;
+                material.SetFloat("_Cutoff", gltfMaterial.alphaCutoff);
+            }
+            else if (gltfMaterial.alphaModeEnum == AlphaMode.BLEND) {
+                material.SetOverrideTag("RenderType", "Transparent");
+                material.SetInt("_SrcBlend", (int)UnityEngine.Rendering.BlendMode.SrcAlpha);
+                material.SetInt("_DstBlend", (int)UnityEngine.Rendering.BlendMode.OneMinusSrcAlpha);
+                material.SetInt("_ZWrite", 0);
+                material.DisableKeyword("_ALPHATEST_ON");
+                material.EnableKeyword("_ALPHABLEND_ON");
+                material.DisableKeyword("_ALPHAPREMULTIPLY_ON");
+                material.renderQueue = (int)UnityEngine.Rendering.RenderQueue.Transparent;
+            }
+            else {
+                material.SetOverrideTag("RenderType", "Opaque");
+                material.SetInt("_SrcBlend", (int)UnityEngine.Rendering.BlendMode.One);
+                material.SetInt("_DstBlend", (int)UnityEngine.Rendering.BlendMode.Zero);
+                material.SetInt("_ZWrite", 1);
+                material.DisableKeyword("_ALPHATEST_ON");
+                material.DisableKeyword("_ALPHABLEND_ON");
+                material.DisableKeyword("_ALPHAPREMULTIPLY_ON");
+                material.renderQueue = -1;
+            }
```

Comment:
This seems partially redundant with `StandardShaderHelper.SetAlphaModeMask` used in line 137 above.

Could you merge this within the mentioned method?